src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h

```diff
@@ -106,46 +109,58 @@ namespace OpenMS
 
```

Comment:
If we stick to the rule of six, don't we have to add move constructor and move assignment operator?

Reply:
I have already added them (c with type Container).